packages/slice-machine/components/ScreenshotChangesModal/index.tsx

```diff
@@ -143,7 +165,11 @@ const ScreenshotChangesModal = ({
   useEffect(() => {
     setVariationSelector(variationSetter(defaultVariationSelector, slices));
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [defaultVariationSelector, isOpen]);
+  }, [
+    defaultVariationSelector?.sliceID,
```

Comment:
You still need that at the end?

Reply:
yes, for whatever reason deep state reloads happen some time to time, resetting the object passed and therefore resetting forcefully the tab